packages/cms/src/api/mortarRoute.js

```diff
@@ -99,7 +99,7 @@ const formatters4eval = (formatters, locale) => formatters.reduce((acc, f) => {
     acc[name] = FUNC.parse({logic: f.logic, vars: ["n"]}, acc, locale);
   }
   catch (e) {
-    if (verbose) console.error("Server-side Malformed Formatter encountered: ", e.message);
+    if (verbose) console.error("Server-side Malformed Formatter encountered: ", e.message, `Formatter name: ${name}`);
```

Comment:
- don't put this behind `verbose` (we will probably always want to show errors on prod servers)